packages/material-ui/src/utils/ownerDocument.ts

```diff
@@ -1,3 +1,3 @@
-export default function ownerDocument(node: Node | undefined): Document {
-  return (node && node.ownerDocument) || document;
-}
+import { ownerDocument } from '@material-ui/utils';
+
+export default ownerDocument;
```

Comment:
nit: I think we should remove these files. As far as I can tell they're not necessary anymore which means we can safely reduce the number of files we ship.

Reply:
We have local imports in all core components to this files, I am re-exporting to avoid changes in all core files. I've started with  that approach here https://github.com/mui-org/material-ui/pull/23203 but there were too many changes

Reply:
Sounds good. Let's keep this conversation open for visibility.